### ISSUE TITLE
Warning: findDOMNode is deprecated in StrictMode.

### ISSUE THREAD

**Issue by bharadhwaj** [react-grid-layout/react-draggable#465]
I am getting an error when I am trying to use Draggable,

> Warning: findDOMNode is deprecated in StrictMode. findDOMNode was passed an instance of Draggable which is inside StrictMode. Instead, add a ref directly to the element you want to reference. Learn more about using refs safely here: https://fb.me/react-strict-mode-find-node

Seems like an error inside the library code for me, and not very sure is there anything I can do to solve the error in my end?

I am using uncontrolled Draggable component. 

Any help is appreciated.

**Comment by ricardo-jsx** [comment]
I believe that this comment may help you: https://github.com/raatmarien/playcardswith.me/issues/56#issuecomment-608322650

TL;DR: This warning comes from `<React.StrictMode />` and, for now, it hasn't caused any unexpected behavior and the only way to remove it is to remove from your project `React.StricMode />` itself.

**Comment by STRML** [comment]
Duplicate of #440. Accepting PRs but I don't have the time to fix this properly yet.
